### PR TITLE
Fetch list pullrequests fix

### PR DESCRIPTION
**Pull request by arabyalhomsi**
I hope this one will be the final! I hope I covered all your comments and provided matching improvments.

### CODE REVIEW

File: backend/services/PullRequest.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          ]
+        })
+      }
+    })
+
+    this.data = prArray
+
+    return this
+  }
+  /**
```
Comment (paulodiovani) [inline backend/services/PullRequest.js:56]: Missing line break.

File: backend/services/PullRequest.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    }
+  }
+
+  /**
+   * Group pull requests by user id
+   */
+  groupByUser () {
+    let prArray = []
+
+    this.data.items.map((item) => {
```
Comment (paulodiovani) [inline backend/services/PullRequest.js]: you only use `.map` if you want it to return a new array.

Otherwise, use `forEach`.

**Review by paulodiovani — CHANGES_REQUESTED** [review]

File: backend/services/PullRequest.js (hunk 1 above)
Comment (arabyalhomsi) [reply backend/services/PullRequest.js:56]: shouldn't these configurations be added in the linter?

**Comment by arabyalhomsi** [comment]
Done @paulodiovani

File: backend/services/PullRequest.js (hunk 1 above)
Comment (paulodiovani) [reply backend/services/PullRequest.js:56]: It could

Comment (arabyalhomsi) [reply backend/services/PullRequest.js:56]: I'll try to add it

File: backend/api/controllers/PullRequestController.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const PullRequest = require('../../services/PullRequest')
+
+const PullRequestController = express()
+
+PullRequestController.get('/:year?', async function (req, res) {
```
Comment (paulodiovani) [inline backend/api/controllers/PullRequestController.js:6]: What the question mark does? :thinking:

**Review by paulodiovani — APPROVED** [review]
Will be merging now.
Check the comment I left, please.

File: backend/api/controllers/PullRequestController.js (hunk 3 above)
Comment (arabyalhomsi) [reply backend/api/controllers/PullRequestController.js:6]: It basically makes the param optional. So, if someone does not enter a year, they will be given the results from the current year.

**Pull request merged**